test/rules/import-statement-format.ts

```diff
@@ -0,0 +1,36 @@
+import assert from 'assert';
+import linter from '../utils/parsers';
+const { times } = require('lodash');
+
+const config = {
+  rules: { 'import-statement-format': 'error' },
```

Comment:
how do we decide whether it is warn or error?

Reply:
this is decided in the rule file